lisp/ess-inf.el

```diff
@@ -125,7 +125,7 @@ If `ess-plain-first-buffername', then initial process is number-free."
                         (= n 1))) ; if not both first and plain-first add number
               (concat ":" (number-to-string n)))))
 
-(defun inferior-ess (&optional ess-start-args customize-alist no-wait)
+(defun inferior-ess (ess-start-args customize-alist &optional no-wait)
```

Comment:
In the other PR I have renamed `ess-start-args` to `start-args`. Similarly I think we can use an abbreviation here, e.g. `cust-alist`. This is easier to read and a clearer signal that it's not a global variable like `ess-r-customize-alist` (since global variables have long names).

Reply:
That's all fine with me. Like I said, I can rebase this once you merge that other PR and resolve the conflicts. :+1:

Reply:
Already merged :)

Reply:
Great, updated here.